cmstestsuite/TestCleanCheckout.py

```diff
@@ -77,7 +77,8 @@ def setup_cms():
     os.environ["PYTHONPATH"] = "%(TEST_DIR)s" % CONFIG
 
     info("Building cms.")
-    sh("./setup.py build")
+    sh("./prerequisites.py compile_l10n")
+    sh("./prerequisites.py compile_isolate")
```

Comment:
I meant we can keep the setup build for these things, instead of adding this new script. Is prerequisite a standard? I feel like setup build is much more common.